docs/pages/enroll-resources/workload-identity/aws-oidc-federation.mdx

```diff
@@ -0,0 +1,431 @@
+---
+title: Configuring Workload Identity and AWS OIDC Federation with JWTs
+description: Configuring AWS to accept Workload Identity JWTs as authentication using OIDC Federation
+---
+
+<Admonition type="tip" title="Preview">
+Teleport Workload Identity is currently in Preview. This means that some
+features may be missing. We're actively looking for design partners to help us
+shape the future of Workload Identity and would love to
+[hear your feedback](mailto:product@goteleport.com).
+</Admonition>
+
+Teleport Workload Identity issues flexible short-lived identities in JWT format.
+AWS OIDC Federation allows you to use these JWTs to authenticate to AWS
+services.
+
+This can be useful in cases where a machine needs to securely authenticate with
+AWS services without the use of a long-lived credential. This is because the
+machine can authenticate with Teleport without using any shared secrets by
+using one of our delegated join methods.
+
+In this guide, we'll configure Teleport Workload Identity and AWS to allow our
+workload to authenticate to the AWS S3 API and upload content to a bucket.
+
+## How it works
+
+This implementation differs from using the Teleport Application Service to protect
+AWS APIs in a few ways:
+
+- Requests to AWS are not proxied through the Teleport Proxy Service, meaning
+  reduced latency but also less visibility, as these requests will not be
+  recorded in Teleport's audit log.
+- Workload Identity works with any AWS client, including the command-line tool
+  but also their SDKs.
+- Using the Teleport Application Service to access AWS does not work with Machine
+  ID and therefore cannot be used when a machine needs to authenticate with AWS.
```

Comment:
I think it would make sense to move this text to the Workload Identity introduction to reduce the length of the non-instructional text in this guide. It's also not specific to the setup shown in this guide. We can then move the "OIDC Federation vs Roles Anywhere" text into this section.